marvelApp/app/src/main/java/com/marvelapp06/marvelapp/quiz/QuizResultActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.marvelapp06.marvelapp.quiz
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.marvelapp06.marvelapp.R
+import kotlinx.android.synthetic.main.activity_quiz_result.*
+
+class QuizResultActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_quiz_result)
+
+        val totalQuestions = intent.getIntExtra(Constants.TOTAL_QUESTIONS, 0)
+        val correctAnswers = intent.getIntExtra(Constants.CORRECT_ANSWERS, 0)
+
+        tv_score.text = "Your Score is $correctAnswers out of $totalQuestions."
```

Comment:
Extrair para string.xml utilizando parâmetros 
ex: `<string name="your_score_is_out_of">Your score is %1$d out of %2$d</string>`

Reply:
Feito